CHANGELOG.md

```diff
@@ -5,64 +5,52 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 
 New Features
 ------------
-
-- The [Dominant Speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker)
+- [Dominant Speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker)
   and [Network Quality](https://www.twilio.com/docs/video/using-network-quality-api)
-  APIs are generally available.
-
-- Previously, Room emitted "reconnecting" and "reconnected" events while recovering
-  from a disruption in your media connection. Now, it will emit these events while
-  recovering from a disruption in your signaling connection as well. As of now, this
-  is an opt-in feature and can be enabled with a temporary ConnectOptions flag as follows:
-
-  ```js
-  const { connect } = require('twilio-video');
-  const room = await connect(token, {
-    _useTwilioConnection: true
-  });
-  room.on('reconnecting', error => {
-    if (error.code === 53001) {
-      console.log('Reconnecting your signaling connection!', error.message);
-    } else if (error.code === 53405) {
-      console.log('Reconnecting your media connection!', error.message);
-    }
-  });
-  ```
-
-  When you opt in for this feature, you join a Room using our new signaling transport,
-  which enables us to detect and recover from disruptions in your signaling connection.
-  Whenever your signaling connection is interrupted, the signaling back-end waits
-  for you to reconnect for a period of 30-45 seconds, before it determines that you
-  have left the Room. As a result, if you close the tab/browser or navigate away from
-  your web application without disconnecting from the Room, the other Participants
-  will only be notified after the reconnecting period is over. So, we recommend that
-  you disconnect from the Room when you detect a tab/browser close or page navigation
-  as follows:
-
-  ```js
-  window.addEventListener('beforeunload', () => {
-    room.disconnect();
-  });
-  ```
+  APIs are now generally available.
+- twilio-video.js now supports versions of Safari that enable Unified Plan as
+  the default SDP format. As of now, Unified Plan is enabled by default in the latest
```

Comment:
As of now ...  -> As of the date of this release ...

Instead of "latest Safari Technology Review", can we add the specific preview version?

Reply:
@aymenn We deliberately decided to not mention the specific release since we don't have a clear answer from Apple about when they are going to turn Unified Plan on.